manual/manual/cmds/top.etex

```diff
@@ -126,6 +126,10 @@ The following command-line options are recognized by the "ocaml" command.
 \begin{unix}
 The following environment variables are also consulted:
 \begin{options}
+\item["OCAMLTOP_INCLUDE_PATH"]  Additional directories to search for
+  compiled object code files (".cmi", ".cmo" and ".cma"). The directory list
+  is searched in left-to-right order.
```

Comment:
I would still add something like "after the include directories specified on the command line via `-I` have been searched." to make that cristal clear.